contrib/local.mk.example

```diff
@@ -9,7 +9,7 @@
 
 # By default, the jemalloc family of memory allocation functions are used.
 # Uncomment the following to instead use libc memory allocation functions:
-# CMAKE_EXTRA_FLAGS="-DUSE_JEMALLOC=OFF"
+# DEPS_CMAKE_FLAGS := -DUSE_BUNDLED_JEMALLOC=OFF
```

Comment:
I see, although I'm still unsure why there's no option to "hard" disable jemalloc (like USE_JEMALLOC=OFF). That way, jemalloc wouldn't be built and you wouldn't need to specify USE_BUNDLED_JEMALLOC=OFF as well as USE_JEMALLOC=OFF.

Anyways, it's a rather small inconsistency so perhaps it could be dealt with later.